crates/starknet-os/src/hints/find_element.rs

```diff
@@ -17,7 +17,7 @@ use crate::utils::get_variable_from_root_exec_scope;
 
 #[rustfmt::skip]
 
-pub const FIND_ELEMENT: &str = indoc! {r#"array_ptr = ids.array_ptr
+pub const SEARCH_SORTED_OPTIMISTIC: &str = indoc! {r#"array_ptr = ids.array_ptr
```

Comment:
I'm not sure I like the name change here. It better describes what the function does, but it deviates from our strategy of reusing naming conventions and code organization from `cairo-lang` (the fn there is called `find_element` inside `find_element.cairo`).

In other words, if someone is trying to compare our code to that of `cairo-lang` then this new name will not help them.

Reply:
But that's the point I'm making, there are three hints in that file, all doing very similar things, but with just small changes between them. This hint is actually the one in `func search_sorted_optimistic`, hence the name change.

Reply:
Oh, I'm sorry, I misunderstood you earlier. Looks good then!